install.js

```diff
@@ -9,6 +9,7 @@ const path = require('path');
 const del = require('del');
 const child_process = require('child_process');
 const os = require('os');
+const findChromeVersion = require('find-chrome-version');
```

Comment:
This package brings in 26 packages ([ref](https://npm.anvaka.com/#/view/2d/find-chrome-version/1.0.2)), that seems quite a lot.

Reply:
Yes, it is a lot, but this is a package that is used for dev almost all the time. I don't like taking in new dependencies, but `request` is worse by far.

Reply:
I just saw a comment on the other issue discussing https://github.com/testimio/chrome-version. It might be a better option. It takes no deps.